src/sentry/templates/sentry/auth_close.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+  <script>
+  // sends message to window that opened this popup if user is authenticated and closes the window
+  function extractDomain(url) {
+    var matches = url.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i);
+    var domain = matches && matches[1];
+    return domain;
+  }
+
+    (function checkAuthChange() {
+      var logged_in = '{{ logged_in }}';
+      if (window.opener && logged_in === 'True') {
+        window.opener.postMessage('User is logged in', "https://" + extractDomain(window.location.href));
```

Comment:
updated and renamed the function - let me know if it covers the needed cases or if I should tweak